test/integration/cdi/example-driver/app/server.go

```diff
@@ -244,13 +245,32 @@ func NewCommand() *cobra.Command {
 	}
 	cmd.AddCommand(controller)
 
+	// kubelet plugin
+	kubeletPluginFlagSets := cliflag.NamedFlagSets{}
+	fs = kubeletPluginFlagSets.FlagSet("kubelet plugin")
+
+	kubeletDir := fs.String("kubelet-dir", "/var/lib/kubelet", "Path to the Kubelet directory.")
```

Comment:
please avoid using hardcoded paths. Either import it from the k8s code or at least define your own constant.

Reply:
Sorry, I think I do not understand what is 'hard code'. Isn't it configurable by users, and the default is /var/lib/kubelet?

I cannot find how to make it configurable.
When I asked before @pohly with the example os.Getenv, you said about 'drivername'.
> 	driverName := fs.String("drivername", "example-driver.cdi.k8s.io", "Resource driver name.")
Isn't it hard-coded as 'example-driver.cdi.k8s.io'?

Reply:
define a constant, e.g. ```const defaultKubeletDir "/var/lib/kubelet"``` and use it in the fs.String call. Or, better, import it from k8s if it exists there.

Reply:
@bart0sh AHA!!!!!! Thanks!